src/Network/NWPathMonitor.cs

```diff
@@ -22,9 +22,17 @@ public class NWPathMonitor : NativeObject {
 		[DllImport (Constants.NetworkLibrary)]
 		extern static IntPtr nw_path_monitor_create ();
 
+		public NWPath currentPath;
```

Comment:
That should be a property (not a field) and named `CurrentPath` (casing is incorrect for .net)

Reply:
You can find more about our coding guidelines here https://www.mono-project.com/community/contributing/coding-guidelines/